upload/common/ingest_notifier.py

```diff
@@ -17,8 +17,18 @@ class IngestNotifier:
     FILE_UPLOAD_EXCHANGE = 'ingest.file.staged.exchange'
     FILE_UPLOADED_QUEUE = 'ingest.file.create.staged'
 
-    def __init__(self):
-        self.id = str(uuid.uuid4())
+    FILE_VALIDATED_EXCHANGE = "ingest.validation.exchange"
```

Comment:
Did you clear this exchange change with the Ingest team?  They gave me this endpoint.